sematic/types/types/image.py

```diff
@@ -0,0 +1,29 @@
+# Standard Library
+import uuid
+from dataclasses import dataclass
+
+# Third-party
+import magic  # type: ignore
+
+# Sematic
+from sematic.types.registry import SummaryOutput, register_to_json_encodable_summary
+
+
+@dataclass
+class Image:
+    bytes: bytes
+
+    @classmethod
+    def from_file(cls, file_path: str) -> "Image":
```

Comment:
Can you add `from_bytes` so you can, for example, easily convert from an in-memory pillow image to Sematic `Image`?

Reply:
Already supported:
```python
image = Image(bytes=image_bytes)
```

In fact, this type is not particularly specific to images. It is called `Image` only so that the UI can have a special display for it.

We could have a base `File` type to store arbitrary files, with a Download button in the UI, then `Image` would simply be a subclass of `File` with a different UI treatment.